src/libs/actions/Task.js

```diff
@@ -260,9 +266,25 @@ function completeTask(taskReportID, taskTitle) {
     ];
 
     // Update optimistic data for parent report action
-    const optimisticParentReportData = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
-    if (!_.isEmpty(optimisticParentReportData)) {
-        optimisticData.push(optimisticParentReportData);
+    const optimisticDataForParentReportAction = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
+    if (!_.isEmpty(optimisticDataForParentReportAction)) {
+        optimisticData.push(optimisticDataForParentReportAction);
+    }
+
+    // Multiple report actions can link to the same child. Both share destination (task parent) and assignee report link to the same report action.
+    // We need to find and update the other parent report action (in assignee report). More info https://github.com/Expensify/App/issues/23920#issuecomment-1663092717
+    const assigneeReportAction = ReportUtils.getTaskParentReportActionIDInAssigneeReport(taskReport);
```

Comment:
And this would correspond to the `11` reportActionID, correct?

Reply:
What do you mean by `11`?

Reply:
coming from here https://github.com/Expensify/App/pull/24110#discussion_r1286680767

Reply:
Please check https://github.com/Expensify/App/pull/24110/files/5152ef3f5c1ed27b7320e470a8114d8dcad81c01#r1286700086

Reply:
This code would get `12`